en/docs/setup/migrating-preparing-for-migration.md

```diff
@@ -142,6 +146,25 @@ Now let's see how to do the blue-green deployment with WSO2 Identity Server.
         defaultAutoCommit="false"
 
         ```
+
+    ??? tip "A sample configuration written for the Oracle DB type will look this"
```

Comment:
```suggestion
    ??? tip "Sample configuration written for the Oracle DB type"
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2484/commits/045ea4b59c6d26a54259cca970f1c2f55a866558